<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZhuangbeiArea</class>
 <widget class="QWidget" name="ZhuangbeiArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>166</width>
    <height>205</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="judge0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="judge1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="judge2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="wuqiWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>147</width>
       <height>29</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>147</width>
       <height>29</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="fangjuWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>147</width>
       <height>29</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>147</width>
       <height>29</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="jianmaWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>147</width>
       <height>29</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>147</width>
       <height>29</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="jiamaWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>147</width>
       <height>29</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>147</width>
       <height>29</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
